src/proc_image_processing/cpu/filters/detectors/square_detector.h

```diff
@@ -67,10 +66,14 @@ namespace proc_image_processing {
                     std::vector<cv::Point> approx;
 
                     // test each contour
-                    for (size_t i = 0; i < contours.size(); i++) {
+                    for (auto &contour : contours) {
```

Comment:
Can we get consistent across all file for this (eg. see ln49 of center_coffin_detector.h)

Reply:
Sometimes we can't use those loops since we need the index. In this case, we did not need the index so it's pointless to use one.

Reply:
ha ok I just saw in the call of drawContours(), all good.